### PR TITLE
Agents can reschedule appointments

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: app/views/appointments/reschedule.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<%= render 'shared/required_fields_note' %>
+<%= form_for [@appointment], layout: :basic do |f| %>
+  <div class="row form-group js-appointment-availability-calendar-container">
+    <div class="col-md-12">
+      <h2>Choose available slot <%= required_label %></h2>
+      <p class="text-danger hide" id="slot-error">Please choose a slot</p>
+      <div class="appointment-availability-calendar js-appointment-availability-calendar" data-module="AppointmentAvailabilityCalendar" data-error-id="slot-error" data-events=".slots-data" data-default-date="<%= BookableSlot.next_valid_start_date %>"></div>
+      <input type="hidden" name="available-slots-json" class="slots-data" value="<%= @available_slots_json %>">
+      <div class="hide">
+        <%= f.text_field :start_at, value: nil, data: { 'selected-start': true }, class: 't-start-at' %>
```
Comment (benbarnett) [inline app/views/appointments/reschedule.html.erb]: Can these just be `input type="hidden"`s?

File: app/views/appointments/search.html.erb [hunk 2]
```diff
@@ -56,6 +56,10 @@
             <span class='glyphicon glyphicon-pencil' aria-hidden="true"></span>
             <span class="sr-only">Edit appointment</span>
           <% end %>
+          <%= link_to appointment_reschedule_path(result), title: 'Reschedule appointment', class: 'btn btn-info' do %>
+            <span class='glyphicon glyphicon-calendar' aria-hidden="true"></span>
```
Comment (benbarnett) [inline app/views/appointments/search.html.erb]: Got some single and double quote stuff going on here – should be doubles

File: app/views/appointments/reschedule.html.erb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<div class="page-header">
+  <ol class="breadcrumb">
+    <li><a href="<%= root_path %>">Home</a></li>
+    <li class="active">Reschedule an appointment</li>
+  </ol>
+
+  <h1>Reschedule an appointment</h1>
+</div>
+<%= render 'shared/required_fields_note' %>
```
Comment (benbarnett) [inline app/views/appointments/reschedule.html.erb]: Maybe a cheeky line break or two here?

File: app/views/appointments/reschedule.html.erb (hunk 1 above)
Comment (AndrewVos) [inline app/views/appointments/reschedule.html.erb]: Yep, done.

File: app/views/appointments/reschedule.html.erb (hunk 3 above)
Comment (AndrewVos) [inline app/views/appointments/reschedule.html.erb]: Done

File: app/views/appointments/search.html.erb [hunk 4]
```diff
@@ -53,9 +53,13 @@
         <td><span class="status"><%= result.status %></span></td>
         <td>
```
Comment (benbarnett) [inline app/views/appointments/search.html.erb]: Can we add `<td nowrap="true">` so the buttons sit on a single line here

**Pull request merged**